Makefile

```diff
@@ -20,6 +20,9 @@ deploy: .dapper dapper-image clusters
 $(TARGETS): .dapper dapper-image
 	./.dapper -m bind $@ --k8s_version $(k8s_version) --globalnet $(globalnet) --deploytool $(deploytool)
 
+release: .dapper dapper-image
```

Comment:
It doesn't let me comment up for suggestion, but do this:

```bash
TARGETS := $(shell ls -p scripts | grep -v / | grep -v release)
```
otherwise make will complain (just a warning)